bodhi/server/static/css/site.css

```diff
@@ -285,7 +269,7 @@ ul.updateslist li {
   font-size:3em;
   line-height: 1em;
   font-weight: 200;
-}
+}*/
```

Comment:
Do you want to leave this commented out?

Reply:
Nope! this is not needed anymore. removed it entirely in this updated [commit](https://github.com/fedora-infra/bodhi/pull/3500/commits/2e45e91f18ca0609cc1b3e327434bfdabcecb91d)